ostree-simplified-installer.sh

```diff
@@ -36,6 +36,11 @@ sudo cp files/fdo/rendezvous-server.yml /etc/fdo/rendezvous-server.conf.d/
 sudo cp files/fdo/serviceinfo-api-server.yml /etc/fdo/serviceinfo-api-server.conf.d/
 
 # Install yq to modify service api server config yaml file
+# Workaround - https://issues.redhat.com/browse/RHEL-21528
+if [[ "$ID" == "rhel" ]] && [[ $(echo -e "$VERSION_ID\n8.8"|sort -rV|head -n 1) == "8.8" ]]; then
```

Comment:
@litian1992 Thanks for doing this!
Do not need to check `$ID`, just check `$VERSION_ID`. This issue existed on RHEL 8.6 and 8.8.

Reply:
I have the '$ID' check because I thought Fedora and CentOS would use this script also. Then the next condition asserts version is less or equal to 8.8.

Reply:
This is an example to determine the host:  https://github.com/virt-s1/rhel-edge/blob/main/ostree-simplified-installer.sh#L106